src/tutorials/0004-mutable-file-system/10.md

```diff
@@ -8,23 +8,29 @@ MFS has a [`files.read`](https://github.com/ipfs/interface-js-ipfs-core/blob/mas
 The method takes this format:
 
 ```js
-await ipfs.files.read(path, [options])
+ipfs.files.read(path, [options])
 ```
 
 The `path` provided is the path of the file to read, and it must point to a file rather than a directory.
 
-The `files.read` method returns a Buffer, which can be converted to a string using the method `toString('utf8')`. For example:
+The `files.read` method returns an Async Iterable, that iterates over the file's chunks of data, i.e. Buffers, which can be converted to a string using the method `toString('utf8')`. For example:
 
 ```js
-let bufferedContents = await ipfs.files.read('/directory/some-file.txt')  // a buffer
+const concat = require('it-concat')
+
+let bufferedContents = await concat(ipfs.files.read('/directory/some-file.txt'))  // a buffer
 let contents = bufferedContents.toString('utf8') // a string
 ```
 
 or
 
 ```js
-let contents = (await ipfs.files.read('/directory/some-file.txt')).toString('utf8') // a string
+const concat = require('it-concat')
+
+let contents = (await concat(ipfs.files.read('/directory/some-file.txt'))).toString('utf8') // a string
 // notice the parentheses around the entire await statement
 ```
```

Comment:
Personally I'd remove this second example - it's not super readable and doesn't help with much other than to save on 1 line of code.